lib/errors/validation-error.ts

```diff
@@ -1,64 +1,85 @@
-'use strict';
-
-const BaseError = require('./base-error');
+import BaseError, { ErrorOptions } from './base-error';
+import type { Model } from '../../types/lib/model';
 
 /**
```

Comment:
I have to admit that this change is a bit too much for my current state of mind, but you have my trust.